packages/core/README.md

```diff
@@ -112,8 +112,10 @@ You can also include class themes within your Sass file instead:
 ```scss
 @use '~@onfido/castor';
 
-@include castor.day-class();
-@include castor.night-class();
+:global {
```

Comment:
Original example was not using CSS modules, so I guess you either would need to remove `:global` or inform that this example is for when using CSS modules.

Reply:
My bad here, I kind of interpreted this bit as a continuation of the previous topic. Will remove both instances of `:global`.